pingdom-checks/appeals.ping

```diff
@@ -25,3 +25,24 @@ pingdom save-check \
   -a group="appeals" \
   -a userids_csv="$STATUSPAGE_SANDBOX_APPEALS_STATUS" \
   -a integrationids_csv="$APPEALS_SLACK_ID"
+
+#v1
+pingdom save-check \
+  --template request-with-apikey \
+  -a name=production-appeals-v1-health-check \
+  -a host=api.va.gov \
+  -a url="/services/appeals/v1/healthcheck" \
+  -a apikey="$PRODUCTION_APPEALS_API_KEY" \
+  -a group="appeals" \
+  -a userids_csv="$NO_USERS" \
```

Comment:
This will have not integration with Statuspage and no Slack messages. Are you _sure_ you don't want to send this to Slack?

Reply:
I thought we'd want to do this in 2 parts (like how we did Forms API (#30 and #31))